cupyx/scipy/spatial/delaunay_2d/_kernels.py

```diff
@@ -0,0 +1,2713 @@
+        point_vec, points_idx, pred_consts):
+    ker_init_point_loc_fast = DELAUNAY_MODULE.get_function(
+        'kerInitPointLocationExact')
+    ker_init_point_loc_fast((PRED_N_BLOCKS,), (PRED_BLOCK_SZ,), (
+        vert_tri, n_vert, exact_check, counter, tri, inf_idx,
+        point_vec, points_idx, pred_consts))
+
+
+def vote_for_point(vert_tri, n_vert, tri, vert_circle, tri_circle,
+                   no_sample, inf_idx, points, pred_consts):
+    ker_vote_for_point = DELAUNAY_MODULE.get_function('kerVoteForPoint')
+    ker_vote_for_point((N_BLOCKS,), (BLOCK_SZ,), (
+        vert_tri, n_vert, tri, vert_circle, tri_circle,
+        no_sample, inf_idx, points, pred_consts
+    ))
+
+
+def pick_winner_point(vert_tri, n_vert, vert_circle, tri_circle,
+                      tri_to_vert, no_sample):
+    ker_pick_winner_point = DELAUNAY_MODULE.get_function('kerPickWinnerPoint')
+    ker_pick_winner_point((N_BLOCKS,), (BLOCK_SZ,), (
+        vert_tri, n_vert, vert_circle, tri_circle, tri_to_vert, no_sample))
+
+
+def shift(shift_idx, in_arr, out_arr, type_str):
+    ker_shift = DELAUNAY_MODULE.get_function(f'kerShiftValues<{type_str}>')
+    ker_shift((N_BLOCKS,), (BLOCK_SZ,), (
+        shift_idx, int(shift_idx.shape[0]), in_arr, out_arr))
+
+
+def shift_opp_tri(shift_idx, in_arr, out_arr):
+    ker_shift = DELAUNAY_MODULE.get_function('kerShiftOpp')
+    ker_shift((N_BLOCKS,), (BLOCK_SZ,), (
+        shift_idx, int(shift_idx.shape[0]), in_arr, out_arr))
+
+
+def shift_tri_idx(idx, shift_idx):
+    ker_shift = DELAUNAY_MODULE.get_function('kerShiftTriIdx')
+    ker_shift((N_BLOCKS,), (BLOCK_SZ,), (
+        idx, int(idx.shape[0]), shift_idx))
+
+
+def split_points_fast(vert_tri, tri_to_vert, tri, ins_tri_map,
+                      exact_check, counter, tri_num, ins_tri_num,
+                      inf_idx, points, points_idx, pred_consts):
+    ker_split_points_fast = DELAUNAY_MODULE.get_function('kerSplitPointsFast')
+    ker_split_points_fast((N_BLOCKS,), (BLOCK_SZ,), (
+        vert_tri, int(vert_tri.shape[0]), tri_to_vert, tri, ins_tri_map,
+        exact_check, counter, int(tri_num), int(ins_tri_num), inf_idx, points,
+        points_idx, pred_consts))
+
+
+def split_points_exact(vert_tri, tri_to_vert, tri, ins_tri_map,
+                       exact_check, counter, tri_num, ins_tri_num,
+                       inf_idx, points, points_idx, pred_consts):
+    ker_split_points_exact = DELAUNAY_MODULE.get_function(
+        'kerSplitPointsExactSoS')
+    ker_split_points_exact((PRED_BLOCK_SZ,), (PRED_N_BLOCKS,), (
+        vert_tri, tri_to_vert, tri, ins_tri_map,
+        exact_check, counter, int(tri_num), int(ins_tri_num), inf_idx, points,
+        points_idx, pred_consts))
+
+
+def split_tri(split_tri, tri, tri_opp, tri_info, ins_tri_map, tri_to_vert,
+              tri_num, ins_tri_num):
+    ker_split_tri = DELAUNAY_MODULE.get_function('kerSplitTri')
+    ker_split_tri((N_BLOCKS,), (32,), (
```

Comment:
Is this typo?

```suggestion
    ker_split_tri((N_BLOCKS,), (BLOCK_SZ,), (
```

Reply:
Yes, this is an implementation detail that comes from the reference C++ code